doc/rippled-example.cfg

```diff
@@ -1076,6 +1076,13 @@ file_size_mult=2
 online_delete=2000
 advisory_delete=0
 
+# This is the persistent datastore for shards. Helpful information can be
+# found here: https://ripple.com/build/history-sharding
```

Comment:
Consider adding a suggestion to uncomment the lines below to enable history sharding, maybe something like

```
# The commented lines below show how to configure the persistent datastore for shards.
# Consider uncommenting to enable the feature and contribute to the health of the Ripple network.
# Helpful information on the sharding can be found here: https://ripple.com/build/history-sharding
```

Reply:
@bachase Thanks. As per Arthur's suggestion, uncommented by default.